guide-for-adding-windows-node.md

```diff
@@ -0,0 +1,229 @@
+
+# Adding Windows nodes
```

Comment:
Can you create a new top-level folder named either `docs` or `guides` to put this in?

Reply:
other than this comment LGTM.
This is a good starting point for docs thanks @fabi200123 !